src/uwtools/tests/api/test_sfc_climo_gen.py

```diff
@@ -2,23 +2,36 @@
 
 from unittest.mock import patch
 
-from iotaa import external, task, tasks
+from iotaa import asset, external, task, tasks
 
 from uwtools.api import sfc_climo_gen
 
 
-def test_execute():
+def test_execute(tmp_path):
+    dot = tmp_path / "graph.dot"
     args: dict = {
         "config_file": "config.yaml",
         "batch": False,
         "dry_run": True,
+        "graph_file": dot,
     }
     with patch.object(sfc_climo_gen, "SfcClimoGen") as SfcClimoGen:
         assert sfc_climo_gen.execute(**args, task="foo") is True
+    del args["graph_file"]
     SfcClimoGen.assert_called_once_with(**args)
     SfcClimoGen().foo.assert_called_once_with()
 
 
+def test_graph():
+    @external
+    def ready():
+        yield "ready"
+        yield asset("ready", lambda: True)
+
+    ready()
+    assert sfc_climo_gen.graph().startswith("digraph")
```

Comment:
I had not heard of a `digraph` before. I looked up what it stands for and found it was for a directed graph(which I have heard of). I've never heard it called by that name in case anyone has the same question I do.